sudoku.py

```diff
@@ -92,34 +75,33 @@ def check_duplicates(self):
             Bool: `True` if no duplicates were found. Otherwiser raises `ValueError`.
         """
         for i in range(9):
-            if not self.check_list_duplicates(self.sudoku[i, :]):
```

Comment:
In general, test should return True, when test is passed and False, when it fails.
In this case, test fails when there are duplicates. For this reason, although for a single function this is not intuitive, it is intuitive in general scope and usability.

Reply:
Generally I'd agree with that statement but regarding this exact case, the method name should at least imply what is a failed test, while currently it only states that it checks if there are existing duplicates. When reading the code where the method was used, I was confused that it returned True, when the "check" essentially failed. In my humble opinion, the method was intended to only check what is the situation regarding duplicate values, and it shouldn't be aware of what purpose will that check serve when it is used.
When it's used, it should be clear from the name, what condition it is going to check.

I'd suggest either modifying name to something like `list_contains_unique_values()` or leaving my modification (which I believe is completely valid and more readable).
Waiting for your decision :)